src/main/java/utility/FigureSupplier.java

```diff
@@ -0,0 +1,49 @@
+package utility;
+
+import java.util.Random;
+import model.Color;
+import model.Figure;
+import model.figure.Circle;
+import model.figure.IsoscelesTrapezoid;
+import model.figure.Rectangle;
+import model.figure.RightTriangle;
+import model.figure.Square;
+
+public class FigureSupplier {
+    public static final String DEFAULT_COLOR = "white";
+    public static final int DEFAULT_DATA = 10;
+    public static final int MAX_RANDOM_VALUE = 100;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+    private final Color[] colors = Color.values();
+
+    public Figure getRandomFigure() {
+        int figureType = random.nextInt(colors.length);
```

Comment:
The variable `figureType` should be named to reflect that it is selecting a figure type, not a color. A better name might be `figureIndex` or `figureNumber`.